cdap-docs/_common/common-build.sh

```diff
@@ -279,12 +272,21 @@ function check_includes() {
 }
 
 function download_includes() {
+  local includes_dir=${1}
   # $1 passed is the directory to which the downloaded files are to be written.
   # For an example of over-riding this function, see developer/build.sh
   echo "No includes to be downloaded."
 }
 
-function test_includes () {
+function test_includes_directory() {
+  local includes_dir=${1}
+  if [ ! -d "${includes_dir}" ]; then
```

Comment:
Removed function, as it isn't needed. The directory is always created right before this step, so it is a redundant check.